numpy/core/src/umath/loops.c.src

```diff
@@ -1617,6 +1631,42 @@ NPY_NO_EXPORT void
 }
 /**end repeat1**/
 
+NPY_NO_EXPORT void
+@TYPE@_max_abs(char **args, npy_intp *dimensions, npy_intp *steps, void* NPY_UNUSED(func))
+{
+#if @uint_matched@
+    /* Everything except signbit set, 0x7fffffffffffffff */
+    const @type@ mask0 = -0.0@c@;
```

Comment:
this trick should not be used for the generic code, as a good C citizen we cannot assume anything on the float representation. sse2 code gets away with it as its x86 only.